package.json

```diff
@@ -2,28 +2,30 @@
   "name": "englishprofile-words",
   "version": "0.0.1",
   "description": "",
-  "main": "index.js",
-  "type": "module",
-  "engines": {
-    "node": ">=0.14"
-  },
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "generate-csv": "node --loader ts-node/esm index.ts"
-  },
+  "keywords": [
+    "anki",
+    "englishprofile"
+  ],
   "author": "",
   "license": "ISC",
+  "main": "src/index.ts",
+  "scripts": {
+    "start": "ts-node ./src/index.ts",
```

Comment:
Thanks for your amazing contribution! It's really valuable for our project!

Why have you decided to use `start`? Isn't `run` more common for scripts?

Reply:
I thought `npm start` is common for all the npm packages

Reply:
Tests, build and other scripts are run with `[npm run](https://docs.npmjs.com/cli/v8/commands/npm-run-script)`.

Reply:
You can use `npm run start` as well :)